web-app/src/components/jobs/JobDetailModal.tsx

```diff
@@ -0,0 +1,606 @@
+                                <strong>Industry</strong>
+                            </Col>
+                            <Col>
+                                {jobPost.companyDetail.industry}
+                            </Col>
+                        </Row>
+                        <Row>
+                            <Col md={3}>
+                                <strong>Website</strong>
+                            </Col>
+                            <Col>
+                                {jobPost.companyDetail.companyWebsite}
+                            </Col>
+                        </Row>
+                        <Row>
+                            <Col md={3}>
+                                <strong>Location</strong>
+                            </Col>
+                            <Col>
+                                <JobLocationList>
+                                    {jobPost.companyDetail.location.map((v) => {
+                                        return <JobLocationText key={v}>{v}</JobLocationText>
+                                    })}
+                                </JobLocationList>
+                            </Col>
+                        </Row>
+                    </JobCompanyBasicDetail>
+                    <CompanyRatingsWrapper>
+                        <Row>
+                            <Column md={6}>
+                                <CompanyRating>
+                                    <ListGroup>
+                                        <ListGroup.Item>
+                                            <RatingTitle>
+                                                <StarsIcon />&nbsp;{jobPost.company} Ratings
+                                            </RatingTitle>
+                                        </ListGroup.Item>
+                                        <ListGroup.Item className="highlighted">
+                                            <RatingContainer highlighted>
+                                                <span>Average Rating</span>
+                                                <div>
+                                                    <StarRatings
+                                                        rating={jobPost.companyDetail.ratings.average}
+                                                        starDimension="18px"
+                                                        starSpacing="2px"
+                                                    />
+                                                </div>
+                                                <span>
+                                                    {jobPost.companyDetail.ratings.average}
+                                                </span>
+                                            </RatingContainer>
+                                        </ListGroup.Item>
+                                        <ListGroup.Item className="even" >
+                                            <RatingContainer >
```

Comment:
What is the purpose of using ListGroup.Item and RatingContainer as two separate divs? Don't use two divs where one div could get you the same result. Also, "even" is a confusing name. You can instead use something like "alternative-background"

Reply:
ListGroup.Item is a component provided by react-bootstrap, RatingContainer for styling the inner content of ListGroup.Item,
rather than overwriting library component preferable to have a new style component.